<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockHelp</class>
 <widget class="QDockWidget" name="DockHelp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Documentation</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="9">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="backButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="application.qrc">
          <normaloff>:/images/br_prev.png</normaloff>:/images/br_prev.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="forwardButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="application.qrc">
          <normaloff>:/images/br_next.png</normaloff>:/images/br_next.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QToolButton" name="toggleFindButton">
        <property name="toolTip">
         <string>Search within this page</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="application.qrc">
          <normaloff>:/images/gtk-search.png</normaloff>:/images/gtk-search.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="homeToolButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="application.qrc">
          <normaloff>:/images/go-home-16.png</normaloff>:/images/go-home-16.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Find:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="caseBox">
        <property name="styleSheet">
         <string notr="true">padding-right: 8; padding-left: 8;</string>
        </property>
        <property name="text">
         <string>Match Case </string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="previousFindButton">
        <property name="icon">
         <iconset resource="application.qrc">
          <normaloff>:/images/br_prev.png</normaloff>:/images/br_prev.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="nextFindButton">
        <property name="icon">
         <iconset resource="application.qrc">
          <normaloff>:/images/br_next.png</normaloff>:/images/br_next.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="findLine">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">padding-left: 8px; padding-right: 40px</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QCheckBox" name="wholeWordBox">
        <property name="text">
         <string>Whole words</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="text">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="openLinks">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="nextFindAct">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/br_next.png</normaloff>:/images/br_next.png</iconset>
   </property>
   <property name="text">
    <string>Find Next</string>
   </property>
  </action>
  <action name="findPreviousAct">
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/images/br_prev.png</normaloff>:/images/br_prev.png</iconset>
   </property>
   <property name="text">
    <string>Find Previous</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toggleFindButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>14</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toggleFindButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>findLine</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>11</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toggleFindButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>caseBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>15</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toggleFindButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>wholeWordBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toggleFindButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>nextFindButton</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toggleFindButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>previousFindButton</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
